pkg/apis/build/v1alpha2/image_validation_test.go

```diff
@@ -262,9 +279,9 @@ func testImageValidation(t *testing.T, when spec.G, it spec.S) {
 		it("image.cacheSize has not decreased", func() {
```

Comment:
might be nice to update the text here

Reply:
@tylerphelan Which text? 😕 Sorry...

Reply:
We got it :) Thanks.

Reply:
I actually meant the test text: `it("image.cacheSize has not decreased", func() {` -> `it("image.cache.volume.size has not decreased", func() {` 😀